daemon/containerd/service.go

```diff
@@ -86,10 +87,8 @@ func (i *ImageService) CountImages(ctx context.Context) int {
 }
 
 // CreateLayer creates a filesystem layer for a container.
```

Comment:
I guess the same applies here; if this is creating a snapshot.

Wondering if we should consider removing the abstraction, and instead had separate methods for "Snapshot" vs "graphdriver".